api/HTMLFrameSetElement.json

```diff
@@ -99,10 +99,10 @@
         "__compat": {
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": false
```

Comment:
https://github.com/mdn/browser-compat-data/pull/3732 sets `onstorage` support to true, and that is easily confirmed manually.

It looks like in https://github.com/mdn/browser-compat-data/pull/3645 you're not applying implements statements. Here's the one that matters here:
https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/html/html_frame_set_element.idl?l=45&rcl=aae10192b9f96a766448a4e48570d67c775b9d63